Bottom: 05d35ec8c13c4cfdbff6d8fe1e752db1c844b9e5
Top:    3614b31fd2f9efdff80356860a5b1194c955d033
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-07 07:02:11 -0500

Refresh of mirror

---

diff --git a/farm-checkout b/farm-checkout
index 6a9a51c..4fd7a2f 100755
--- a/farm-checkout
+++ b/farm-checkout
@@ -26,9 +26,10 @@ for tag in $tags; do
 	   git clone --quiet --local --mirror $linuxgit $gittree &&
 	   git --bare --git-dir=$gittree config core.bare false &&
 	   git --bare --git-dir=$gittree config core.worktree $srctree &&
+	   mkdir $srctree &&
 	   cd $srctree &&
-	   git --bare --git-dir=$gittree --work-tree=$srctree checkout --quiet $tag &&
-	   echo "gitdir: $gittree" >$srctree/.git; then
+	   echo "gitdir: $gittree" >.git &&
+	   git checkout --quiet $tag; then
 		echo -e "\e[32mOK\e[0m"
 	else
 		error=1
